gitlab/props.conf

```diff
@@ -45,7 +45,7 @@ EVAL-release_status = if((object_kind=release AND isnotnull('action')), 'action'
 EVAL-release_url = if(isnotnull('url'), 'url', null())
 EVAL-review_author_login = if(('object_kind'="note" AND 'object_attributes.noteable_type'="MergeRequest" AND isnotnull('object_attributes.change_position.position_type')), 'user.username', null())
 EVAL-review_state = if('object_attributes.action'="approved", "approved", if(('object_attributes.noteable_type'="MergeRequest" AND isnotnull('object_attributes.change_position')), "commented", null()))
-EVAL-latest_commit_hash = if(isnotnull(mvindex('commits{}.id', -1)), mvindex('commits{}.id', -1) , null())
+EVAL-latest_commit_hash = if(isnotnull(mvindex('commits{}.id', -1)), mvindex('commits{}.id', -1), if((isnotnull('object_attributes.sha') AND 'object_kind'="pipeline"), 'object_attributes.sha', if((isnotnull('sha') AND 'object_kind'="build"), 'sha', null())))
```

Comment:
More of a Splunk/SPL question:
Should I put the non-`mvindex` statements at the end of the eval? Thus not ever hitting that `mvindex` when it matches on the more simple pipeline/build stuff?